src/parser/jobs/rdm/Gauge.js

```diff
@@ -0,0 +1,199 @@
+				// this.cooldowns.resetCooldown(ACTIONS.CORPS_A_CORPS.id)
+				// this.cooldowns.resetCooldown(ACTIONS.DISPLACEMENT.id)
+				isCalculated = true
+			}
+
+			//Determine if the ability we used should yield any mana gain.
+			const manaGain = MANA_GAIN[abilityId] ? {...MANA_GAIN[abilityId]} : null
+			if (manaGain) {
+				isCalculated = true
+			}
+
+			if (abilityId === ACTIONS.SCATTER.id && isCalculated) {
+				//Check the Buffs on the player for Enhanced scatter, if so gain goes from 3 to 8
+				if (this.combatants.selected.hasStatus(STATUSES.ENHANCED_SCATTER.id)) {
+					manaGain.white = 8
+					manaGain.black = 8
+				}
+			}
+			if (isCalculated) {
+				if (manaGain) {
+					//console.log(`Ability: ${getAction(abilityId).name} White Gain: ${manaGain.white||0} Black Gain: ${manaGain.black||0}`)
+				}
+				if (manaGain && manaGain.white && this._blackMana - this._whiteMana > 30) {
+					//If we have more than 30 black mana over White, our White gains are halved
+					//console.log('Black Imbalance')
+					this._whiteManaLostToImbalance += Math.ceil(manaGain.white/2)
+					//console.log(`White Lost: ${this._whiteManaLostToImbalance}`)
+					manaGain.white = Math.floor(manaGain.white/2)
+				}
+
+				if (manaGain && manaGain.black && this._whiteMana - this._blackMana > 30) {
+					//If we have more than 30 white mana over black, our black gains are halved
+					this._blackManaLostToImbalance += Math.ceil(manaGain.black/2)
+					manaGain.black = Math.floor(manaGain.black/2)
+				}
+
+				if (manaGain) {
+					this._whiteMana += manaGain.white||0
+					this._blackMana += manaGain.black||0
+				}
+
+				if (this._whiteMana > 100) {
+					this._whiteManaWasted += this._whiteMana - 100
+					if (manaGain) {
+						this._whiteOverallManaGained += (manaGain.white - (this._whiteMana - 100))
+					}
+					this._whiteMana = 100
+				} else if (manaGain) {
+					this._whiteOverallManaGained += manaGain.white||0
+				}
+
+				if (this._blackMana > 100) {
+					this._blackManaWasted += this._blackMana - 100
+					if (manaGain) {
+						this._blackOverallManaGained += (manaGain.black - (this._blackMana - 100))
+					}
+					this._blackMana = 100
+				} else if (manaGain) {
+					this._blackOverallManaGained += manaGain.black||0
+				}
+			}
+
+			return
+		}
+
+		_onComplete() {
+			if (this._whiteManaWasted && this._whiteManaWasted > 0) {
+				this.suggestions.add(new Suggestion({
+					icon: ACTIONS.VERHOLY.icon,
+					content: <Fragment>
+										Ensure you don&#39;t overcap your White Mana before a combo, you lost {this._whiteManaWasted} White Mana
+					</Fragment>,
+					//severity: SEVERITY.MEDIUM,
+					severity: this._whiteManaWasted > 80 ? SEVERITY.MAJOR : this._whiteManaWasted > 20 ? SEVERITY.MEDIUM : SEVERITY.MINOR,
+					why: <Fragment>
+										Overcapping White Mana indicates your balance was off, and you lost out on Enchanted Combo damage.  You should look to execute at 80/80 or as close to it as possible.
```

Comment:
The `content` and `why` of a few of these are visa-versa. I'll write some docs tomorrow or something, but in general:

`content` should explain what you should/shouldn't be doing in concrete terms, and maybe explain why that is the case.

`why` should be explaining why _this suggestion_ is showing up.

Using this suggestion as an example, you could use a combo of the first half of `content` and most of `why` as the `content`, then have `why` as something like `You lost (...) white mana due to capped gauge resources.` or something similar.